src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4235,7 +4235,9 @@ .FORWARDHELPCATEGORY Cmdlet
     else
     {
         # Respect PAGER, use more on Windows, and use less on Linux
-        $moreCommand,$moreArgs = $env:PAGER -split '\s+'
+        $path = split-path $env:PAGER
+        $file,$moreArgs = split-path $env:PAGER -leaf | % {$_ -split '\s+'}
+        $moreCommand = join-path $path $file
```

Comment:
This doesn't work with my current `$env:PAGER` value e.g. `$env:PAGER = 'less.exe "-PsPage %db?B of %D:.\. Press h for help or q to quit\.$"'`

Reply:
I would suggest starting with @mklement0's proposal https://github.com/PowerShell/PowerShell/issues/8130#issuecomment-433987035 and adding tests

Reply:
Thanks, @SteveL-MSFT. Note that in order to embed my code from the linked comment in the verbatim string in the C# code the `"` instances must be doubled.